Solution1/BusinessLayer/Service/Abstract/IApiService.cs

```diff
@@ -0,0 +1,14 @@
+﻿using DataAccessLayer.Model;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace BusinessLayer.Service.Abstract
+{
+    public interface IApiService
```

Comment:
give more meaningful name - IWeatherApiService for example, what if you will request multiple apis? not just api

Reply:
+